From d099685b1b297c8c8016a464cac2375e6ebcd762 Mon Sep 17 00:00:00 2001
Message-Id: <d099685b1b297c8c8016a464cac2375e6ebcd762@dist-git>
From: Michal Privoznik <mprivozn@redhat.com>
Date: Tue, 25 Apr 2017 17:38:46 +0200
Subject: [PATCH] qemuDomainAttachNetDevice: Support attach of type="user"

RHEL-7.4: https://bugzilla.redhat.com/show_bug.cgi?id=1420668
RHEL-7.3.z: https://bugzilla.redhat.com/show_bug.cgi?id=1445231

This has worked in previous releases.
My commit c266b60440 broke it.

Signed-off-by: Michal Privoznik <mprivozn@redhat.com>
(cherry picked from commit 9d87f769726bd5714eb6a930379d8b4c53311196)
Signed-off-by: Michal Privoznik <mprivozn@redhat.com>
Signed-off-by: Jiri Denemark <jdenemar@redhat.com>
---
 src/qemu/qemu_hotplug.c | 3 +++
 1 file changed, 3 insertions(+)

diff --git a/src/qemu/qemu_hotplug.c b/src/qemu/qemu_hotplug.c
index b7302a5f9..df4a67ea3 100644
--- a/src/qemu/qemu_hotplug.c
+++ b/src/qemu/qemu_hotplug.c
@@ -1063,6 +1063,9 @@ qemuDomainAttachNetDevice(virQEMUDriverPtr driver,
         break;
 
     case VIR_DOMAIN_NET_TYPE_USER:
+        /* No preparation needed. */
+        break;
+
     case VIR_DOMAIN_NET_TYPE_SERVER:
     case VIR_DOMAIN_NET_TYPE_CLIENT:
     case VIR_DOMAIN_NET_TYPE_MCAST:
-- 
2.12.2

